packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { GetLimits } from '@deriv/api-types';
+import { DesktopWrapper, Text } from '@deriv/components';
+import { getCurrencyDisplayCode, getPlatformSettings } from '@deriv/shared';
+import { TReactChildren } from 'Types';
+
+type TAccountTransferBullet = {
+    children: TReactChildren;
```

Comment:
@hamid-deriv Why not `React.ReactNode` or even use `React.PropsWithChildren`? 🤔

Reply:
`TReactChildren` is the alias for `React.ReactNode` which is defined in the cashier types.

I think `React.PropsWithChildren` is used for the type when we want to implement `children` alongside other props. And as this is just children, I think `ReactNode` is a better type for this.